### PR TITLE
Support for new ignored_alerts.txt GTFS file

### PR DESCRIPTION
**Pull request by lboyarsky**
This goes with https://github.com/mbta/OpenTripPlanner/pull/22

It updates the OBA library to the latest version (all commits except last two), fixes the merge conflicts (penultimate commit) and adds support for new `ignored_alerts.txt` (last commit).

### CODE REVIEW

File: onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/IgnoredAlert.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  @CsvField(optional = true)
+  private String description;
+
+  public IgnoredAlert() {
+
+  }
+
+  public IgnoredAlert(IgnoredAlert a) {
+    this.id = a.id;
+    this.description = a.description;
```
Comment (paulswartz) [inline onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/IgnoredAlert.java:23]: Do we care about parsing the description? Does it get used?

Comment (lboyarsky) [reply onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/IgnoredAlert.java:23]: No we're not using it at the moment but I just wanted our changes to be consistent with existing codebase (where they parse all the fields) so they look less like a hack.

**Review by paulswartz — APPROVED** [review]
🍰

**Comment by handorff** [comment]
✅ 

Didn't look at the new OBA stuff, but your changes lgtm.

**Review by handorff — APPROVED** [review]

**Pull request merged**